### PR TITLE
Remove signature store from registry.

### PR DESCRIPTION
**Pull request by RichardScothern**
Return a generated signature for manifest pull.

Signed-off-by: Richard Scothern <richard.scothern@docker.com>

### CODE REVIEW

**Comment by RichardScothern** [comment]
@dmcgowan

**Comment by dmcgowan** [comment]
Code changes LGTM

Need to discuss how this should be documented and possible impact. I expect it to be low impact since the signatures were never integrated into any flow used by private registries, but external tools could have used them in ways unknown to us. The deprecation documentation should also point to using notary as an alternative for signing registry content.

**Comment by RichardScothern** [comment]
docs added @dmcgowan

File: registry/handlers/app.go [hunk 1]
```diff
@@ -178,16 +180,10 @@ func NewApp(ctx context.Context, config *configuration.Configuration) *App {
 	}
 
 	options := registrymiddleware.GetRegistryOptions()
```
Comment (dmcgowan) [inline registry/handlers/app.go]: `registry/handlers/app.go:182: no new variables on left side of :=`

**Comment by dmcgowan** [comment]
LGTM

**Pull request merged**